docs/scripts/create-config-examples.js

```diff
@@ -5,24 +5,39 @@ const TOML = require('@iarna/toml');
 const YAML = require('yaml');
 
 // Helper functions
-const getExampleValue = (param, deepFilter) => {
-  let value;
+const getArrayValue = (obj) => {
+  const enumVal = (obj.enum != null) ? [Object.keys(obj.enum)[0]] : null;
 
-  const getArrayValue = (obj) => {
-    const enumVal = (obj.enum != null) ? [Object.keys(obj.enum)[0]] : null;
+  const examplesVal = (obj.examples != null && obj.examples.length > 0) ? [obj.examples[0]] : null;
 
-    const examplesVal = (obj.examples != null && obj.examples.length > 0) ? [obj.examples[0]] : null;
+  return obj.default || examplesVal || enumVal || null;
+}
 
-    return obj.default || examplesVal || enumVal || null;
-  }
+const getValue = (obj) => {
+  const enumVal = (obj.enum != null) ? Object.keys(obj.enum)[0] : null;
 
-  const getValue = (obj) => {
-    const enumVal = (obj.enum != null) ? Object.keys(obj.enum)[0] : null;
+  const examplesVal = (obj.examples != null && obj.examples.length > 0) ? obj.examples[0] : null;
 
-    const examplesVal = (obj.examples != null && obj.examples.length > 0) ? obj.examples[0] : null;
+  return obj.default || examplesVal || enumVal || null;
+}
 
-    return obj.default || examplesVal || enumVal || null;
-  }
+// Convert object to TOML string
+const toToml = (obj) => {
+  return TOML.stringify(obj);
+}
+
+// Convert object to YAML string
+const toYaml = (obj) => {
+  return `---\n${YAML.stringify(obj)}`;
```

Comment:
One thing I'm slightly curious about is if we can control that conversion a little more granularly - ex for the `source` key in `remap` using a block literal is nicer looking: 

```yaml
source: |
  #all my vrl things
```